README.md

```diff
@@ -1,8 +1,16 @@
 # Virtual MMX
 
 [![](https://img.shields.io/github/issues/wintergatan-community/virtual-mmx)](https://github.com/wintergatan-community/virtual-mmx/issues)
- [![](https://img.shields.io/github/issues-pr/wintergatan-community/virtual-mmx)](https://github.com/wintergatan-community/virtual-mmx/pulls)
- [![](https://img.shields.io/badge/contribute-info-blue?style=flat-square)](https://github.com/wintergatan-community/virtual-mmx/blob/master/CONTRIBUTING.md)
- [![Discord](https://img.shields.io/badge/Discord-join-7289DA?logo=discord&style=flat-square)](https://discord.gg/rMK6DFT)
+[![](https://img.shields.io/github/issues-pr/wintergatan-community/virtual-mmx)](https://github.com/wintergatan-community/virtual-mmx/pulls)
+[![](https://img.shields.io/badge/contribute-info-blue?style=flat-square)](https://github.com/wintergatan-community/virtual-mmx/blob/master/CONTRIBUTING.md)
+[![Discord](https://img.shields.io/badge/Discord-join-7289DA?logo=discord&style=flat-square)](https://discord.gg/rMK6DFT)
 
-A WebApp to program songs on a virtual Marble Machine X
+## About
+
+A WebApp to program songs on a virtual Marble Machine X. Songs can be exported and stored on local disk.
```

Comment:
Perhaps mention that future features can be seen on this board https://github.com/orgs/wintergatan-community/projects/1

Reply:
Yes, that will also be helpfull. Added.